src/debug/seer-integration.js

```diff
@@ -75,6 +75,14 @@ export const layerEditListener = cb => {
   });
 };
 
+export const removeLayer = id => {
+  if (!window.__SEER_INITIALIZED__ || !id) {
```

Comment:
Idea: Maybe this "uglyness" (of looking at variables on the window) could be hidden in a function `isSeerInitialized()`? Initially here in deck.gl, and later provided by the seer API?